irobotclient/test/test_irobot_configuration_handler.py

```diff
@@ -0,0 +1,157 @@
+import unittest
+from unittest.mock import MagicMock
+
+import errno
+import argparse
+import os
+
+from irobotclient import configuration_handler
+from irobotclient.custom_exceptions import IrobotClientException
+
+class TestConfigurationSetup(unittest.TestCase):
+
+    def setUp(self):
+        self._parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter)
```

Comment:
I've changed the tests to use an `argparse.Namespace` object passed into the individual functions and an argument list passed into the overall `configuration_handler.run(args)` function.  Can you check it out @colin-nolan and tell me what you think?